docs/examples/gnoi/gnoic/index.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: default
+title: "gNOIc examples"
+date: 2021-07-16 12:17:00 --0600
+categories:
+---
+
+## About gRPC
+
+gRPC - Google Remote Procedure Call
+
+gRPC uses protobuf and HTTP/2
+
+## About gNOI
+
+gNOI - gRPC Network Operations Interface
+
+gNOI defines a set of gRPC-based microservices for executing operational commands on network devices.
+
+gNOI repository https://github.com/openconfig/gnoi
+
+As example, this gNOI proto file https://github.com/openconfig/gnoi/blob/master/system/system.proto defines the service `System` with the RPC `Traceroute` and `Ping`
+
+- Ping executes the ping command on the target and streams back the results
+- Traceroute executes the traceroute command on the target and streams back the results
+- As you can see in the proto file, the field VRF is not defined for these messages
+
+## About gNOI support on EOS
+
+https://eos.arista.com/eos-4-24-2f/gnoi/
+
+Examples:
```

Comment:
Need a newline in between 32 and the list starting on 33 for the list to render correctly

Reply:
done, thanks